soc/nordic/nrf54h/bicr/Kconfig

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2024 Nordic Semiconductor
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_NRF54H20_GENERATE_BICR
+	bool "Generate nRF54H20 BICR file"
+	depends on SOC_NRF54H20_CPUAPP
```

Comment:
Could it be made to not depend on `SOC_NRF54H20_CPUAPP` specifically? Since BICR is required in most cases, it is useful to be able to generate BICR even not compiling for cpuapp.